.github/workflows/build-gem.yml

```diff
@@ -0,0 +1,165 @@
+name: Build gem
+
+on:
+  workflow_dispatch:
+    inputs:
+      push:
+        description: Push gem
```

Comment:
where does this input come from? under what conditions the gem is pushed to github packages?

Reply:
`workflow_dispatch` means that it can be manually triggered from Github UI